packages/scheduled-tasks/src/lib/ScheduledTaskHandler.ts

```diff
@@ -0,0 +1,63 @@
+import { container } from '@sapphire/framework';
+import type { Awaited } from '@sapphire/utilities';
+import { ScheduledTaskRedisStrategy } from './strategies/ScheduledTaskRedisStrategy';
+import { ScheduledTaskStore } from './structures/ScheduledTaskStore';
+import type { ScheduledTasksOptions, ScheduledTasksTaskOptions, ScheduledTaskStrategy } from './types';
+
+export class ScheduledTaskHandler {
+	public readonly strategy: ScheduledTaskStrategy;
+
+	public constructor(options: ScheduledTasksOptions | undefined) {
+		this.strategy = options?.strategy ?? new ScheduledTaskRedisStrategy();
+		void this.strategy.connect();
```

Comment:
While I get why, `void`ing this means users won't know why tasks don't work if redis fails to connect. :c

Reply:
This should be done in a place where we can await, not inside a constructor.